components/datetime/src/neo.rs

```diff
@@ -425,7 +425,7 @@ size_test!(
 pub struct DateTimeFormatter<FSet: DateTimeNamesMarker> {
     pub(crate) selection: DateTimeZonePatternSelectionData,
     pub(crate) names: RawDateTimeNames<FSet>,
-    pub(crate) calendar: UntaggedFormattableAnyCalendar,
+    pub(crate) calendar: FormattableAnyCalendar,
```

Comment:
Observation: I think I had introduced `UntaggedFormattableAnyCalendar` because it reduced the stack size of `DateTimeFormatter`. But, the tests that enforce stack size are still passing, so maybe there's a new niche where the extra field can fit now.